src/tests/suite/daos_obj_ec.c

```diff
@@ -285,9 +287,73 @@ ec_rec_list_punch(void **state)
 	ioreq_fini(&req);
 }
 
+static void
+ec_agg_check_replica_on_parity(test_arg_t *arg, daos_obj_id_t oid, char *dkey,
+			       char *akey, daos_off_t offset, daos_size_t size,
+			       bool exist)
+{
+	d_sg_list_t	sgl;
+	d_iov_t		sg_iov;
+	d_iov_t		dkey_iov;
+	daos_iod_t	iod = { 0 };
+	daos_recx_t	recx;
+	daos_handle_t	oh;
+	char		*buf;
+	struct daos_oclass_attr *oca;
+	uint64_t	shard;
+	int		rc;
+
+	rc = daos_obj_open(arg->coh, oid, 0, &oh, NULL);
+	assert_rc_equal(rc, 0);
+
+	/** init dkey */
+	d_iov_set(&dkey_iov, dkey, strlen(dkey));
+
+	/** init scatter/gather */
+	buf = (char *)malloc(size);
+	assert_true(buf != NULL);
+	d_iov_set(&sg_iov, buf, size);
+	sgl.sg_nr = 1;
+	sgl.sg_nr_out = 0;
+	sgl.sg_iovs = &sg_iov;
+
+	/** init I/O descriptor */
+	d_iov_set(&iod.iod_name, akey, strlen(akey));
+	iod.iod_nr	= 1;
+	iod.iod_size	= 1;
+	recx.rx_idx	= offset;
+	recx.rx_nr	= size;
+	iod.iod_recxs	= &recx;
+	iod.iod_type	= DAOS_IOD_ARRAY;
+
+	daos_obj_verify(arg->coh, oid, DAOS_EPOCH_MAX);
```

Comment:
assume it is only to sync DTX status for the obj

Reply:
yes, correct.

Reply:
all tests passed, there is only some build failure due to environment.